doc/src/contributing.rst

```diff
@@ -79,10 +78,16 @@ IMAPClient (provided these versions of Python are installed!).
 .. _`tox`: http://testrun.org/tox/
 
 Writing Unit Tests
-==================
+~~~~~~~~~~~~~~~~~~
 Protocol level unit tests should not act against a real IMAP server
 but should use canned data instead. The IMAPClientTest base class
 should typically be used as the base class for any tests - it provides
 a mock IMAPClient instance at `self.client`. See the tests in
 imapclient/tests/test_imapclient.py for examples of how to write unit
 tests using this approach.
+
+
+Authors and acknowledgment
+--------------------------
+
+.. include:: ../../AUTHORS.rst
```

Comment:
I wonder if this section should be it's own top level section in index.rst instead of being here.